tests/Platforms/PostgreSQLPlatformTest.php

```diff
@@ -891,7 +891,6 @@ public function testAltersTableColumnCommentIfRequiredByType(): void
         self::assertSame(
             [
                 'ALTER TABLE "foo" ALTER "bar" TYPE TIMESTAMP(0) WITHOUT TIME ZONE',
-                'ALTER TABLE "foo" ALTER "bar" DROP DEFAULT',
```

Comment:
There's no reason to expect this statement. It's probably expected because at the time of writing the test the schema comparison already worked like this.